tests/test_routes.py

```diff
@@ -177,3 +178,106 @@ def test_get_wishlist_list(self):
         data = resp.get_json()
         self.assertEqual(len(data), 5)
         self.assertEqual(data, wishlist_array)
+
+    ######################################################################
+    #  W I S H L I S T   I T E M   T E S T   C A S E S   H E R E
+    ######################################################################
+    def test_create_wishlist_item(self):
+        """It should create a new Wishlist Item and associate it with a specific Wishlist"""
+        # Create a Wishlist to associate the item with
+        wishlist = WishlistFactory()
+        resp_wishlist = self.client.post(BASE_URL, json=wishlist.serialize())
+        self.assertEqual(resp_wishlist.status_code, status.HTTP_201_CREATED)
+        wishlist_data = resp_wishlist.get_json()
+
+        # Create a Wishlist Item
+        wishlist_item = WishlistItemFactory()
+        resp = self.client.post(
+            f'{BASE_URL}/{wishlist_data["id"]}/items',
+            json=wishlist_item.serialize(),
+            content_type="application/json"
+        )
+        self.assertEqual(resp.status_code, status.HTTP_201_CREATED)
+
```

Comment:
I see a check that the Location header is set, but we could go one step forward and make sure that it is what we expect it to be by comparing against an f-string that constructs it given the wishlist id and the newly assigned wishlist_item.id.  (You should also include a test that confirms that the wishlist.id is != None.)

Reply:
Tests added.